peripheral.joystick/resources/buttonmaps/xml/linux/Wireless_Controller_13b_8a.xml

```diff
@@ -1,7 +1,10 @@
 <?xml version="1.0" ?>
 <buttonmap>
     <device name="Wireless Controller" provider="linux" buttoncount="13" axiscount="8">
-        <configuration />
+        <configuration>
+            <axis index="2" center="-1" range="1" />
+            <axis index="5" center="-1" range="1" />
```

Comment:
It looks like the triggers got wrongly detected. Can you reset this `<configuration>` block?